rotkehlchen/assets/exchanges_mappings/okx.py

```diff
@@ -72,4 +72,5 @@
     evm_address_to_identifier('0x088cd8f5eF3652623c22D48b1605DCfE860Cd704', ChainID.ARBITRUM_ONE, EvmTokenKind.ERC20): 'VELA',  # noqa: E501
     evm_address_to_identifier('0xBBc2AE13b23d715c30720F079fcd9B4a74093505', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'ERN',  # noqa: E501
     evm_address_to_identifier('0xf091867EC603A6628eD83D274E835539D82e9cc8', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'ZETA',  # noqa: E501
+    evm_address_to_identifier('0xCa14007Eff0dB1f8135f4C25B34De49AB0d42766', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'STRK',  # noqa: E501
```

Comment:
instead of doing this in every file why not put it in `COMMON_ASSETS_MAPPINGS` ?